src/styles/card.scss

```
@@ -0,0 +1,117 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid var(--border-color);
+  background: var(--button-color-hover);
+  transition: all 0.3s ease-in-out;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```